tests/test_odgovlt.py

```diff
@@ -195,3 +197,48 @@ def test_OdgovltHarvester(app, db, mocker):
     assert result['report_status'] == 'added'
     assert result['errors'] == []
     assert was_last_job_considered_error_free()
+    ckanapi = CkanAPI()
+    ids = ckanapi.package_list()
+    package = [ckanapi.package_show(id=ids[0])]
+    assert [(x['title'],) for x in package] == [
+        ('2014 m. vidutinio metinio paros '
+         'eismo intensyvumo duomenys',),
+    ]
+    assert [(x['notes'],) for x in package] == [
+        ('Pateikiama informacija: kelio numeris, ruožo pradžia, '
+         'ruožo pabaiga, vidutinis metinis paros eismo '
+         'intensyvumas, metai, automobilių tipai',),
+    ]
```

Comment:
The maximum line length is 120 characters, so your lines can be longer.

https://github.com/ivpk/opendata.gov.lt-mysql-import/blob/master/.editorconfig#L13